examples/hyperbee.js

```diff
@@ -0,0 +1,160 @@
+'use strict'
+
+process.env.DEBUG = 'bfx:*'
+
+const path = require('path')
+
+const async = require('async')
+const hypercore = require('hypercore')
+const Hyperbee = require('hyperbee')
+
+const replicate = require('@hyperswarm/replicator')
+
+const { Candle } = require('bfx-api-node-models')
+const { SYMBOLS, TIME_FRAMES } = require('bfx-hf-util')
+const EMAStrategy = require('bfx-hf-strategy/examples/ema_cross')
+
+const { onCandle, onStart, onEnd } = require('../lib/ws_events')
+const initState = require('../lib/init_state')
+
+const hopts = {
+  overwrite: true
+}
+
+const hbOpts = {
+  valueEncoding: 'json'
+}
+
+const rawCandleData = require('./btc_candle_data.json')
+
+rawCandleData.sort((a, b) => {
+  return a[0] - b[0]
+})
+
+const kp = 'candles!5m!'
```

Comment:
Would be cool to use the bfx-hf-strategy key function here:

```
const HFS = require('bfx-hf-strategy')
const candleKey = HFS.candleMarketDataKey({
  symbol: SYMBOLS.BTC_USD,
  tf: TIME_FRAMES.ONE_HOUR
})
```

Reply:
good idea! done!